Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
     "components/plurals",
     "components/provider_cldr",
     "components/provider_fs",
+    "components/provider_ppucd",
```

Comment:
Should it be `ppucd` or `ucd`? The data is UCD, and `PP` is just an artifact. In my mind it's more like `JSON CLDR` vs` XML CLDR` - it's still CLDR.

Reply:
Actually I consider this crate to be more like `provider_icu4c`, since ppucd is shipped as part of the ICU4C data package.  The other offline provider, `provider_cldr`, likewise pulls data from CLDR JSON.

Reply:
Good question, the analogy is very apt.

Reply:
2021-01-28: Keep `provider_ppucd` for now until we can come up with a better name.